OxfordBodleian/Juel-Jensen/BDLaethg28.xml

```diff
@@ -33,39 +35,44 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
-                          <hi resp="rubric">ሚካኤል፡</hi> ስሙዕ፡ አጸን<gap reason="illegible"/> ሠናይ፡ ፄና፡ መዓዛ፡ <cb n="2v"/> በጺሐከ፡ ግብተ፡ ለልብየ፡
-                          አመ፡ ትካዛ፡ ለነፍሰ፡ ዚአየ፡ በት<add place="above">ን</add>ባሌከ፡ ፡ ናዝዛ፡ እንበለ፡ ጊዜ፡ <gap reason="illegible"/>ላህ፡ ኢይርሳዕ፡ ወሬዛ፡
-                      </incipit>
-                            
-                            <explicit xml:lang="gez">
+
+                            <incipit xml:lang="gez"><!-- I would provide only the first stanza in the printed catalogue.-->
+                                <locus from="2r" to="2v"/>
+                                <hi rend="rubric">ሰላም፡ ለዝክረ፡ ስምከ፡ ምስለ፡ ስመ፡ ልዑል፡ ዘተ</hi>ሳተፈ፡ ወልደ፡
+                                ያሬድ፡ ሄኖክ<hi rend="rubric">፨</hi> በከመ፡ ጸሐፈ፡ ረዳኤ፡ ምንዳባን፡ <hi
+                                    rend="rubric">ሚካኤል፡</hi> ዘከመ፡ ታለምድ፡ ዘልፈ<supplied
+                                    reason="omitted" resp="DR">፡</supplied> ለረዲኦትየ፡ ሰፊሐከ፡ ክንፈ፡ <hi
+                                    rend="rubric">ሰላም<supplied resp="DR" reason="omitted"
+                                        >፡</supplied></hi> ለስእርተ፡ ርእስከ፡ ዘኢይጤየቅ፡ ግዕዛ፡ <hi
+                                    resp="rubric">ሚካኤል፡</hi> ሳውዕ፡ አጸን<supplied reason="lost"
+                                    resp="SH">ሐ</supplied>ሄ፡ ሠናይ፡ ፄና፡ መዓዛ፡ <cb n="2v"/> በጺሐከ፡ ግብተ፡
+                                ለልብየ፡ አመ፡ ትካዛ፡ ለነፍሰ፡ ዚአየ፡ በት<add place="above">ን</add>ባሌከ፡ ፡ ናዝዛ፡
+                                እንበለ፡ ጊዜ፡ <supplied reason="lost" resp="SH">በ</supplied>ላህ፡ ኢይርሳዕ፡
+                                ወሬዛ፡</incipit>
+
+                            <explicit xml:lang="gez"><!-- I would provide only the last stanza in the printed catalogue.-->
                                 <locus from="15r" to="15v"/>
-                                <hi rend="rubric">ሰላም</hi>፡ ሰላም፡ ለመልክእከ፡ ኵሎን፡ በበአስማቲሆን፡ <hi rend="rubric">ሚካኤል፡</hi> ስርግው፡ በልብሰ፡ ብርሃን፡
-                                ሠናይ፡ ተአርኮትከ፡ በኵሉ፡ ዘመን። ሰራየ፡ ኃጢት፡ እንተ፡ አርከ፡ ነፍሰ፡ ምዕመን<supplied reason="omitted" resp="DR">፡</supplied>
-                                <hi rend="rubric">አምኃ</hi>፡ ሰላም፡ አቅረብኩ፡ ለመልክዕከ፡ ኵሉ፡
-                                ለለ<hi rend="rubric">፩፩</hi>ዘበበክፍሉ፡ <hi rend="rubric">ሚካኤል፡</hi> ክቡር፡ ለልዑል፡ መልአከ፡ ኃይሉ፡ ተወኪፈከ፡ አምኃየ፡
-                                እ<pb n="15v"/>ምኖኃ፡ ሰማያት፡ ዘላእሉ፡ ዕሤተ፡ ጸሎትየ፡ ፈሩ፡ ወአሰብየ፡ ድሉ፡ ለገብርከ፡
-                                <hi rend="rubric">ወልደ፡ <del rend="erasure"/></hi>
+                                <hi rend="rubric">ሰላም</hi>፡ ሰላም፡ ለመልክእከ፡ ኵሎን፡ በበአስማቲሆን፡ <hi
+                                    rend="rubric">ሚካኤል፡</hi> ስርግው፡ በልብሰ፡ ብርሃን፡ ሠናይ፡ ተአርኮትከ፡ በኵሉ፡
+                                ዘመን። ሰራየ፡ ኃጢ <supplied reason="omitted" resp="SH">አ</supplied>ት፡
+                                እንተ፡ አርከ፡ ነፍሰ፡ ምዕመን<supplied reason="omitted" resp="DR">፡</supplied>
+                                <hi rend="rubric">አምኃ</hi>፡ ሰላም፡ አቅረብኩ፡ ለመልክዕከ፡ ኵሉ፡ ለለ ፩፩ ዘበበክፍሉ፡
```

Comment:
Do we also place space _before_ numerals?

Reply:
I don't know, I delete it.